index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+                context flag</a> set:
+                <ol>
+                  1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  User Interaction task source</a>, given this, to perform the following steps:
+                  <ol type='i'>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      Return <a>a promise rejected with</a> a "{{SecurityError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                <dl data-dfn-for="PointerLockOptions" data-link-for=
+                "PointerLockOptions">
+                  If if <a>unadjustedMovement</a> is true and the platform does not 
+                  support <a>unadjustedMovement</a>:
+                  <ol>
+                    1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                    Queue an element task</a> on the 
+                    <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                    User Interaction task source</a>, given this, to perform the following steps:
+                    <ol type='i'>
+                      <li>
+                        <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                        <a>pointerlockerror</a> at <a>this</a>'s 
+                        <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                      </li>
+                      <li>
+                        Return <a>a promise rejected with</a> a "{{NotSupportedError}}" {{DOMException}}.
+                      </li>
+                    </ol>
+                  </ol>
+                </dl>
+                <aside class="note" data-link-for="Document">
+                  "{{NotSupportedError}}" {{DOMException}} should be reserved for 
+                  unsupported options in future iterations of this spec. That is,
+                  no other error conditions should use "{{NotSupportedError}}", so
+                  that web developers can use it for feature testing.
+                </aside>
+              </li>
+              <li>
+                <dl data-dfn-for="PointerLockOptions" data-link-for=
+                "PointerLockOptions">
+                  If none of the above errors have occured the user agent will enter 
```

Comment:
"the user agent will" is strange passive voice phrasing. You're the spec; you say what to do! So just say "1. Enter pointer lock state. 2. Set the user agent's pointer-lock target to this. 3. Set the user agent's pointer-lock options to |options|. 4. Queue an element task..."